app/src/main/res/drawable/splash_background.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <item android:drawable="@color/dark_background_color" />
+    <item
+        android:drawable="@color/light_youtube_primary_color"/>
 
-</layer-list>
+    <item>
+        <bitmap
+            android:src="@drawable/ic_newpipe_triangle_white"
+            android:gravity="center_vertical|center_horizontal" />
+    </item>
+</layer-list>
```

Comment:
Could you explain the changes in this file?

Reply:
The app has two splashes: this one and splash for API >= 23.
Before this change this splash was completely black because the one for new APIs doesn't work correctly here - API < 23 doesn't support setting layer-list's item size.
So when I launched the app on my slow Lollipop phone I thought that the phone froze, but it turned out to be a black NewPipe's Splash Screen.
I changed the background to red (that used in the newer splash screen) and logo `item` to `bitmap`, which shows correctly.